vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/TreeGridElement.java

```diff
@@ -318,4 +318,15 @@ protected boolean isLoading() {
     public int getRowCount() {
         return super.getRowCount();
     }
+
+    /**
+     * Scrolls the TreeGrid to the end. After this the row count is correct.
```

Comment:
Let's remove any assumptions from this PR that scrolling will result in a correct row count. As I elaborated in detail in https://github.com/vaadin/flow-components/issues/7293#issuecomment-2785715664 this can never work reliably.

Reply:
Done